helpers/prepare_aip_cohort.py

```diff
@@ -283,7 +283,7 @@ def main(
     logging.info('Pulling all pedigree members')
     pedigree_dicts, ext_lookup = get_pedigree_for_project(project=project)
 
-    # endpoint gives list of tuples e.g. [['A1234567_proband', 'CPG12341']]
+    # endpoint gives list of tuples e.g. [['A1234567_proband', 'CPT12341']]
```

Comment:
You may want to keep the CPG prefix, and make it `CPGaaaa` so it's clear what the second option is here.